### DISCUSSION TITLE
what is the URL to my Budibase db for POST data?

### DISCUSSION THREAD

**Discussion by ctam19** [Budibase/budibase#15028] [Help] [answered]
Hi everyone,

I am trying to POST data to my Budibase DB. I tried following the Public API page (https://docs.budibase.com/docs/public-api) but I am stuck.  It says " "Current value" of the baseUrl variable to be wherever the Budibase instance you wish to target is hosted"

I set up a free Budibase account. I just need access to one DB. Whenever I use Postman to test the connection, the error msg is Method Not Allowed.

I suspect is that I don't have the right URl set up. Can someone give me an example of how the URL should look like? Or where I can find the URL for my Budibase instance is hosted in.

Thanks!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @ctam19,

I think this might be useful for you: https://docs.budibase.com/reference/rowupdate. You can get your API key from the main apps screen by going to the top right of the screen, clicking on your user, and then clicking the `View API key` in the dropdown menu.

<img width="1466" alt="Screenshot 2024-11-26 at 11 35 09" src="https://github.com/user-attachments/assets/c9b9fca3-3362-4759-8f24-b26bcd53915c">

To get the URL, go into the app you wish and copy the appID. It normally looks something like this: `app_dev_randomString`. If you wish only to hit the published endpoint of the app, remove the _dev.

The table ID can also be found in the URL whenever viewing a table in the data area. It normally looks like `ta_randomString`.

Hopefully this helps.

**Reply by ctam19** [reply]
Thank you.
